src/mscorlib/src/System/DateTime.cs

```diff
@@ -676,8 +671,7 @@ private static long TimeToTicks(int hour, int minute, int second)
         public static DateTime FromOADate(double d) {
             return new DateTime(DoubleDateToTicks(d), DateTimeKind.Unspecified);
         }        
-        
-#if FEATURE_SERIALIZATION
+
         [System.Security.SecurityCritical /*auto-generated_required*/]
         void ISerializable.GetObjectData(SerializationInfo info, StreamingContext context) {
```

Comment:
It is brought down automatically due to inheritance, but I can explicitly add it if you think it helps